apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -86,6 +98,11 @@ loop(Socket, Transport, State) ->
             lager:error("[tcp:loop] Error: ~p~n", [E]),
             ok = Transport:close(Socket)
```

Comment:
Remove the dynamic function call on line 99. Only modules that define callbacks should make dynamic calls.